src/databricks/labs/ucx/queries/progress/main/02_4_migration_status_by_owner_overview.sql

```diff
@@ -0,0 +1,15 @@
+/* --title 'Overview' --description 'Tables and views migration' --width 5 */
+WITH migration_statuses AS (
+    SELECT *
+    FROM ucx_catalog.multiworkspace.latest_historical_per_workspace
+    WHERE object_type = "migration_status"
+)
+
+SELECT
+    owner,
+    100 * COUNT_IF(SIZE(failures) = 0) / SUM(COUNT(*)) OVER (partition by owner) AS percentage,
```

Comment:
```suggestion
    100 * COUNT_IF(SIZE(failures) = 0) / SUM(COUNT(*)) OVER (PARTITION BY owner) AS percentage,
```
nit: `PARTITION BY` are keywords